.github/workflows/ruby-cross-compile.yml

```diff
@@ -0,0 +1,129 @@
+---
+name: Build Ruby gems
```

Comment:
How does this fit in with ruby testing, such as:

```
 - name: Run tests
        run: npm run with-server test:ruby
```

Can this be ported and make this file the canonical ruby testing and building file?